src/app/shared/services/env/env.service.ts

```diff
@@ -21,7 +21,7 @@ export class EnvService {
   blockExplorerUrl: string = environment.blockExplorerUrl;
   SENTRY_ENVIRONMENT: string = environment.SENTRY_ENVIRONMENT;
   INFURA_PROJECT_ID: string =
-    INFURA_PROJECT_ID ?? environment?.INFURA_PROJECT_ID;
+    INFURA_PROJECT_ID ?? process?.env?.INFURA_PROJECT_ID;
   INFURA_PROJECT_SECRET: string =
-    INFURA_PROJECT_SECRET ?? environment?.INFURA_PROJECT_SECRET;
+    INFURA_PROJECT_SECRET ?? process?.env?.INFURA_PROJECT_SECRET;
```

Comment:
Do we also need to change the GHA jobs or will the secrets also be available from `process.env` (it seems like they were previously available via `environment`)?

Reply:
No need to change GHA jobs. This process.env is available locally and it is reading .env file. On GHA there is no file like this, so this part is not changed at all.